app.rb

```diff
@@ -0,0 +1,32 @@
+require 'sinatra/base'
+require_relative './lib/message_manager'
+require_relative './lib/user'
+require './database_connection_setup'
+
+class ChitterApp < Sinatra::Base
+  enable :sessions
+
+  get '/chitter' do
+    @user = User.find(id: session[:user_id])
+    @chitter = Message.show_peeps
```

Comment:
Why not have your program objects have the same name as you DB tables? 

i.e. Peeps table -> Peep object
      Users table -> Users object

You can then have clearer method names like:

`Peeps.all`